### ISSUE TITLE
Acs Patching Support

### ISSUE THREAD

**Issue by Spamzboi123** [uwzis/GPU-Passthrough-Manager#20]
Add the ability to acs patching as some gpu and audio drivers dont have their IOMMU Groups separated.

**Comment by uwzis** [comment]
Doesn't that require downloading a external patched kernel image? This might be better done by the user since this is making serious changes to the operating system. I will be adding support to load the vfio driver to other devices soon.

**Comment by Spamzboi123** [comment]
> Doesn't that require downloading a external patched kernel image? This might be better done by the user since this is making serious changes to the operating system. I will be adding support to load the vfio driver to other devices soon.

yeah that's another way about doing it but you can also try baking it into the operating system

**Comment by uwzis** [comment]
I don't think it would be a good idea to implement it in this project since ACS patching is supposed to be a last option fix that has a lot of potential to brick the OS. This project is just for setting up simple GPU Passthrough and to quickly switch in between drivers. You can feel free to add a PR that adds that functionality, but for the meantime ACS patching will not come to GPUPM.

**Comment by Spamzboi123** [comment]
> I don't think it would be a good idea to implement it in this project since ACS patching is supposed to be a last option fix that has a lot of potential to brick the OS. This project is just for setting up simple GPU Passthrough and to quickly switch in between drivers. You can feel free to add a PR that adds that functionality, but for the meantime ACS patching will not come to GPUPM.

what about instead of baking it in the OS you create a different .img file and the user run it from the bootloader alot of the times when i do Acs patching it does not brick the computer because i did this to 30 computers and i have not had any issues

**Comment by uwzis** [comment]
ACS patching is coming in a new update. It will be accessed through a settings GUI I will be adding ASAP.

**Comment by Spamzboi123** [comment]
> ACS patching is coming in a new update. It will be accessed through a settings GUI I will be adding ASAP.

**Comment by Spamzboi123** [comment]
> ACS patching is coming in a new update. It will be accessed through a settings GUI I will be adding ASAP.

thanks much appreciated

**Comment by Spamzboi123** [comment]
> ACS patching is coming in a new update. It will be accessed through a settings GUI I will be adding ASAP.

did you add ACS patching

**Comment by uwzis** [comment]
I haven't had time to add it. I will be able to in the next couple months.
